examples/07-TwinBuilder/04-Static_ROM_Creation_And_Visualization.py

```diff
@@ -0,0 +1,178 @@
+# Select version and set launch options
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+# Select the Twin Builder version and set launch options. The following code
+# launches Twin Builder 2023 R1 in graphical mode.
+#
+# You can change the Boolean parameter ``non_graphical`` to ``True`` to launch
+# Twin Builder in non-graphical mode. You can also change the Boolean parameter
+# ``new_thread`` to ``False`` to launch Twin Builder in an existing AEDT session
+# if one is running.
+
+desktop_version = "2023.1"
+non_graphical = os.getenv("PYAEDT_NON_GRAPHICAL", "False").lower() in ("true", "1", "t")
+new_thread = True
+
+###############################################################################
+# Set up input data
+# ~~~~~~~~~~~~~~~~~
+# Define needed file name
+
+source_snapshot_data_zipfilename = "Ex1_Fluent_StaticRom.zip"
+source_build_conf_file = "SROMbuild.conf"
+source_props_conf_file = "SROM_props.conf"
+
+# Download data from example_data repository
+temp_folder = generate_unique_folder_name()
+source_data_folder = examples.download_twin_builder_data(source_snapshot_data_zipfilename, True, temp_folder)
+source_data_folder = examples.download_twin_builder_data(source_build_conf_file, True, temp_folder)
+source_data_folder = examples.download_twin_builder_data(source_props_conf_file, True, temp_folder)
+
+# Uncomment the following line for local testing 
+# source_data_folder = "D:\\Scratch\\TempStatic"
+
+data_folder = os.path.join(source_data_folder, "Ex04")
+
+# Unzip training data and config file
+examples.unzip(os.path.join(source_data_folder ,source_snapshot_data_zipfilename), data_folder)
+shutil.copyfile(os.path.join(source_data_folder ,source_build_conf_file), os.path.join(data_folder,source_build_conf_file))
+shutil.copyfile(os.path.join(source_data_folder ,source_props_conf_file), os.path.join(data_folder,source_props_conf_file))
+
+###############################################################################
+# Launch Twin Builder and build ROM component
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+# Launch Twin Builder using an implicit declaration and add a new design with
+# a default setup for building the static ROM component.
+
+tb = TwinBuilder(projectname=generate_unique_project_name(),specified_version=desktop_version, non_graphical=non_graphical, new_desktop_session=new_thread)
+
+# Get the static ROM builder object
+rom_manager = tb._odesign.GetROMManager()
+static_rom_builder = rom_manager.GetStaticROMBuilder()
+
+# Build the static ROM with specified configuration file
+confpath = os.path.join(data_folder,source_build_conf_file)
+static_rom_builder.Build(confpath.replace('\\', '/'))
+
+# Test if ROM was created sucessfully
+static_rom_path = os.path.join(data_folder,'StaticRom.rom')
+if os.path.exists(static_rom_path):
+	tb._odesign.AddMessage("Info","path exists: {}".format(static_rom_path.replace('\\', '/')), "")
+else:
+	tb._odesign.AddMessage("Info","path does not exist: {}".format(static_rom_path), "")
```

Comment:
Here you could use the logger if you want. 
```suggestion
	tb.logger.info("path exists: {}".format(static_rom_path.replace('\\', '/')), "")
else:
	tb.logger.error("path does not exist: {}".format(static_rom_path), "")
```